cinema_service/settings.py

```diff
@@ -10,19 +10,20 @@
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 
+from datetime import timedelta
 from pathlib import Path
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+SECRET_KEY = ("django-insecure-"
+              "6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3")
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
DEBUG is set to True, which is not recommended for production environments as it can expose sensitive information. Ensure this is set to False in production.